proto/Makefile

```diff
@@ -26,9 +26,17 @@ clean:
 	rm -fR gen
 	rm -fR dist
 
+.PHONY: deps
+deps: go-deps py-deps
+
+.PHONY: go-deps
+go-deps:
```

Comment:
should we manage everything + versions via go.mod/sum? I feel uncomfortable with `go install` here

Reply:
I share your discomfort and I suspect this is hiding problems, but I can't find any specific issue at this time. Either way, go.mod is managed automatically with tooling. If `go mod tidy` doesn't this it needs to add it to go.mod, adding it manually won't make things better.

Reply:
same as py deps

should be solved once we move to use buf to generate the bindings,